backend/codesirius/problems/views/language.py

```
@@ -0,0 +1,149 @@
+        """
+        Partially update a language by its primary key
+        """
+        logger.info(f"Partially updating language with ID: {pk}")
+        try:
+            language = Language.objects.get(pk=pk)
+            logger.info(f"Language with ID: {pk} fetched successfully")
+        except Language.DoesNotExist:
+            logger.warning(f"Language with ID: {pk} not found")
+            raise NotFound(f"Language with ID: {pk} not found")
+        serializer = LanguageSerializer(
+            instance=language, data=request.data, partial=True
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)